templates/hadoop-master/files/etc/supervisor/conf.d/flink-history-server.conf

```diff
@@ -10,4 +10,5 @@ autostart=true
 
 environment=
     JAVA_HOME=/opt/openjdk-8,
+    FLINK_CONF_DIR=/etc/flink/conf_history_server,
```

Comment:
is it possible to share the FLINK_CONF_DIR with other Flink components?

Reply:
Flink cannot set separate keytab and principal for history server, which means that the kerberos configurations we added applies to all flink jobs.

Reply:
For other jobs, we can use tgt cache without additional configurations.

Reply:
> Flink cannot set separate keytab and principal for history server

an awkward design ...

Reply:
can we minimize the configuration to keep only history server related items?

Reply:
The Flink community somedays may add the HS dedicated kerberos configuration items then we can merge them back.

Reply:
Makes sense, I'll try it later.